app/models/entry.server.ts

```diff
@@ -50,6 +50,29 @@ export function getEntries(skip?: number, take: number = 20) {
   })
 }
 
+export function getEntriesByInitialLettersAndPage(
+  initialLetters: string,
+  page: string
+) {
+  const pageNumber: number = parseInt(page);
```

Comment:
One more thing — let's make sure your Prettier / ESLint config is working properly, and that it's applying on save. In my workspace, I have it configured so that when I save a file, it automatically gets formatted. And one of the formatting steps is for semicolons to be eliminated. I've noticed a few semicolons here; we just want to make sure the code is formatted consistently between our workspaces.

We could maybe do a quick call to verify your setup.

Reply:
done